packages/venia-concept/src/components/CreateAccountPage/asyncActions.js

```diff
@@ -0,0 +1,9 @@
+import { handleCreateAccount } from '../../actions/user/asyncActions';
+
+export const createAccount = ({ accountInfo, history }) => async dispatch => {
+    try {
+        await dispatch(handleCreateAccount(accountInfo));
+
+        history.goBack();
+    } catch (e) {}
```

Comment:
Please add a comment explaining why this code swallows errors as well.

Reply:
I have added a comment